src/util/Util.js

```diff
@@ -103,13 +103,14 @@ define(['okta', 'util/Logger'], function (Okta, Logger) {
 
   // Trigger an afterError event
   Util.triggerAfterError = function (controller, err) {
-    if (!err.statusCode) {
+    if (err.xhr && err.xhr.status) {
       // Bring the statusCode to the top-level of the Error
       err.statusCode = err.xhr && err.xhr.status;
```

Comment:
- what if `err.statusCode` is already exists, will it be override unexpectedly or not?
- probably just do `statusCode = err.xhr.status`

Reply:
Good question. `statusCode` isn't a top-level key in any of our errors, so we do not need to worry about an overwrite.

Will update line `108` to copy over `err.xhr.status` since we're checking above for the presence of `xhr`.